pkg/base/helm.go

```diff
@@ -34,6 +35,11 @@ func RenderHelm(u *upstreamtypes.Upstream, renderOptions *RenderOptions) (*Base,
 			}
 		}
 
+		// check chart.yaml for Helm version if a helm version has not been explicitly provided
+		if strings.EqualFold(fileName, "Chart.yaml") && renderOptions.HelmVersion == "" {
+			renderOptions.HelmVersion, err = checkChartForVersion(&file)
```

Comment:
this `err` is never checked.  but also, how about making `checkChartForVersion` return "v2" instead of error?  also I think we need to log info level message with helm version used for rendering.

Reply:
I don't suspect we'll hit chart.yaml files that can't be unmarshaled, but it will be easier to troubleshoot if we ever hit that case and we can find an error explaining why it defaulted to v2.  How about the following?
``` Go
if strings.EqualFold(fileName, "Chart.yaml") && renderOptions.HelmVersion == "" {
	renderOptions.HelmVersion, err = checkChartForVersion(&file)
	if err != nil {
		renderOptions.Log.Errorf("could not determine helm version (will use helm v2 by default): %v", err)
	} else {
		renderOptions.Log.Info("rendering with Helm %v", renderOptions.HelmVersion)
	}
}
```

Reply:
I think it should be info in both cases.  If it's an error, it should fail.

Reply:
switched to log.Info for both cases